ixwebsocket/IXSocketTLSOptions.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *  IXSocketTLSOptions.h
+ *  Author: Benjamin Sergeant
+ *  Copyright (c) 2017-2018 Machine Zone, Inc. All rights reserved.
+ */
+
+#pragma once
+
+#include <string>
+
+namespace ix
+{
+    struct SocketTLSOptions
```

Comment:
I have an idea, we can add an empty constructor which will just assert that IXWEBSOCKET_USE_TLS is on

#ifndef IXWEBSOCKET_USE_TLS
assert(false && "To use TLS feature the library must be compiled with USE_TLS);
#endif // something like that ?

Reply:
sounds good; did that